OpenLIFUProtocolConfig/OpenLIFUProtocolConfig.py

```diff
@@ -108,6 +109,67 @@ class OpenLIFUProtocolConfigParameterNode:
 
     """
 
+#
+# OpenLIFUProtocolConfigDialogs
+#
+
+class ProtocolSelectionFromDatabaseDialog(qt.QDialog):
```

Comment:
can you make this dialog start out a bit wider? wide enough to fit a typical protocol name and id entry